app/src/main/java/jet/FriendJet.java

```diff
@@ -0,0 +1,76 @@
+package jet;
+
+import android.graphics.Paint;
+
+import hacking.to.the.gate.Bullet;
+import hacking.to.the.gate.Hittable;
+import hacking.to.the.gate.Velocity;
+
+/**
+ * Created by Ruiqian on 9/20/2015.
+ */
+public class FriendJet extends Jet implements Hittable{
+
+    public static int ATTACK_DAMAGE = 45;
+    public static class Builder extends Jet.Builder {
+        public Builder(float x, float y, float r,int animationType) {
+            super(x,y,r,animationType);
+        }
+        public Builder health(int val){
+            mHealth = val;
+            return this;
+        }
+
+        public Builder maxSpeed(int val){
+            mMaxSpeed = val;
+            return this;
+        }
+
+        public Builder hasDestination(boolean val){
+            mHasDestination = val;
+            return this;
+        }
+
+        public Builder setGunType(int val){
+            mGunType = val;
+            return this;
+        }
+
+        public Builder setBulletStyle(int val){
+            mBulletStyle = val;
+            return this;
+        }
+
```

Comment:
Too many duplicate codes in builder. Should be able to put them into parent class.